src/main/fc/fc_msp.c

```diff
@@ -1302,6 +1303,14 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
 #endif
         break;
 
+    case MSP2_INAV_OUTPUT_MAPPING:
+        for (uint8_t i = 0; i < USABLE_TIMER_CHANNEL_COUNT; ++i) {
+            uint8_t output_usage_flags = timerHardware[i].usageFlags & (TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_MOTOR | TIM_USE_FW_SERVO);
+            if (output_usage_flags)
+                sbufWriteU8(dst, output_usage_flags);
```

Comment:
Shouldn't we output something for every timer output or at least output also timer number?

Reply:
`TIM_USE_ANY` is a special value (zero). It's used in softserial and ledstrip code.

Reply:
probably it is a good idea to send `TIM_USE_ANY` as well. Not required for the direct purpose of this frame, but we can have an issue or ordering in some theorethical setups.

Reply:
Yep, I think we should expose all defined timer channels regardless of their flags.

Reply:
I will propose changes in the next few days to try to address this the best way possible.